DZSwiftUI/DZSwiftUI/Model/Notifications.swift

```diff
@@ -0,0 +1,14 @@
+// Notifications.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Настройки уведомлений
+struct Notifications: Identifiable {
+    // MARK: - Public Properties
+
+    let id = UUID()
+    let title: String
+    let subtitle: String
+    var isToggleOn: Bool
```

Comment:
```suggestion
/// дока
    let id = UUID()
/// дока
    let title: String
/// дока
    let subtitle: String
/// дока
    var isToggleOn: Bool
```
В остальных моделях тоже добавь